src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/dto/json/OppdaterVilkårsvurderingJson.kt

```diff
@@ -0,0 +1,56 @@
+package no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.json
+
+import no.nav.tilleggsstonader.sak.infrastruktur.exception.brukerfeilHvis
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Delvilkår
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.domain.Vurdering
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.DelvilkårDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.dto.tilDto
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.hovedreglerPassBarn
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.RegelId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.regler.SvarId
+import no.nav.tilleggsstonader.sak.vilkår.stønadsvilkår.vilkårsreglerPassBarn
+import java.util.UUID
+
+data class OppdaterVilkårsvurderingJson(
+    val id: UUID,
+    val behandlingId: UUID,
+    val vurdering: List<VurderingJson>,
+)
+
+data class VurderingJson(
+    val regel: RegelId,
+    val svar: SvarId?,
+    val begrunnelse: String?,
+)
+
+fun VurderingJson.tilDomene(): Vurdering = Vurdering(
+    regelId = this.regel,
+    svar = this.svar,
+    begrunnelse = this.begrunnelse,
+)
+
+fun List<VurderingJson>.tilDelvilkårDtoer(): List<DelvilkårDto> {
```

Comment:
Denne fungerer nå kun for pass barn. Det er ikke alltid man skal laga saker som fungerer for allt, men hvis man sender inn `Stønadstype` til denne metoden, så kan den finne ut hvilke hovedregler/stønadsregler man skal bruke. Så har man ikke streng kobling mot pass barn. 
Stønadstypen burde hentes fra behandlingen. Og det tenker jeg er naturlig å gjøre i servicen.

Reply:
Enig 👍 Fikser

Reply:
Ble mye bedre nå 😁